lib/escape.rb

```diff
@@ -0,0 +1,7 @@
+module Helpers
```

Comment:
Modules are expected to reflect folder structure, so you'd put this file lib/helpers/escape.rb`